### DISCUSSION TITLE
Revenir sur ses branches récentes avec git | Vincent Hardouin

### DISCUSSION THREAD

**Discussion by giscus** [VincentHardouin/blog#29] [Announcements]
# Revenir sur ses branches récentes avec git | Vincent Hardouin

Blog de tutorials, humeurs, … sur des sujets tech.

https://vincenthardouin.dev/posts/revenir-sur-ses-branches-recentes-avec-git/

**Comment by jbarotin** [comment]
Merci pour le partage, testé et ajouté dans mon bashrc !
